package.json

```diff
@@ -31,16 +33,18 @@
   },
   "license": "ICS",
```

Comment:
should be `MIT` or `Apache-2.0` unless we have a good reason to use ISC.  based on our conversation, I don't think we do :D